docs/product-manuals/clients/cli-client/index.md

```diff
@@ -54,7 +54,7 @@ Available Commands:
   cancel      Cancel resource
   complete    Complete a resource
   create      Create resources
-  deploy      Creates new workflow defined by provided BPMN or YAML file as workflowPath
+  deploy      Creates new workflow defined by provided BPMN file as workflowPath
```

Comment:
As far as I know there is a deliberate renaming under way which replaces all instances of `workflow` with `process`. I am not sure whether this applies only to the code or also to the documentation.

See e.g. https://github.com/zeebe-io/zeebe/pull/6455

@menski can say more about this and give some context